domains/zeto/pkg/types/config.go

```diff
@@ -91,7 +92,17 @@ var DomainInstanceConfigABI = &abi.ParameterArray{
 }
 
 // marks the version of the Zeto transaction data schema
-var ZetoTransactionData_V0 = ethtypes.MustNewHexBytes0xPrefix("0x00010000")
+var ZetoTransactionDataID_V0 = ethtypes.MustNewHexBytes0xPrefix("0x00010000")
```

Comment:
You could argue that this should be incremented, since I have changed the data format. However, I'm not sure how we feel about backwards compatibility at this staged.